app/controllers/settings/ldap_test_controller.rb

```diff
@@ -0,0 +1,136 @@
+require "net/ldap"
+require "json"
+require "openssl"
+require "base64"
+
+# LdapTestController is used to validate LDAP Connector before saving
+class Settings::LdapTestController < ApplicationController
+  def ldap_filter_search(ldap, baseDN, filter_string)
+    filter = Net::LDAP::Filter.construct(filter_string)
+    attrs = ["mail", "cn", "sn", "objectclass"]
+    search_object = ldap.search(
+      base:          baseDN,
+      filter:        filter,
+      attributes:    attrs,
+      return_result: true
+    )
+    search_object
+  end
+
+  def render_result(result_hash, status, message)
+    result_hash[:result] = { status: status, message: message }
+    render json: result_hash
+  end
+
+  # Returns an ldap object
+  def get_ldap(host, port, start_tls, pem_string, anon_bind, dN, pass)
+    tls = if start_tls == "true"
```

Comment:
Recommend using the ternary operator here. It is more confusing the way it is written here.

Reply:
This was a rubocop recommendation.